### PR TITLE
Service impl

### PR DESCRIPTION
**Pull request by carlosroman**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -74,7 +75,7 @@ func main() {
```
Comment (tosan88) [inline main.go:75]: The `tmeTaxonomyName` should be PN for people. Can you please change it?

Comment (tosan88) [reply main.go:75]: Also do we really want to log the sensitive information? Those will end up in splunk...

File: people/service.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+		if err != nil {
+			log.Errorf("Error while creating OrgService: [%v]", err.Error())
+		}
+	}(s)
+	return s
+}
+
+func (s *peopleServiceImpl) isInitialised() bool {
+	s.RLock()
+	defer s.RUnlock()
```
Comment (tosan88) [inline people/service.go:53]: Nice!

File: people/service.go [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+	return s.initialised
+}
+
+func (s *peopleServiceImpl) shutdown() error {
+	log.Info("Shutingdowwn...")
```
Comment (tosan88) [inline people/service.go]: Typo? :)

File: people/service.go [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+	if err != nil {
+		log.Errorf("ERROR reading from cache file for [%v]: %v", uuid, err.Error())
+		return person{}, false, err
+	}
+	if len(cachedValue) == 0 {
+		log.Infof("INFO No cached value for [%v]", uuid)
+		return person{}, false, nil
+	}
+	var cachedOrg person
```
Comment (tosan88) [inline people/service.go]: cachedPerson?

File: people/service.go [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+	err = json.Unmarshal(cachedValue, &cachedOrg)
+	if err != nil {
+		log.Errorf("ERROR unmarshalling cached value for [%v]: %v", uuid, err.Error())
+		return person{}, true, err
+	}
+	return cachedOrg, true, nil
+
+}
+
+func (s *peopleServiceImpl) newInit() error {
```
Comment (tosan88) [inline people/service.go]: why is the `new` prefix added? I'm just thinking if we ever want to reuse the method e.g. for reloading endpoint....

File: people/service.go [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+	c := make(chan []person)
+	go s.processPeople(c)
+	s.Lock()
```
Comment (tosan88) [inline people/service.go:104]: Also nice here too!

File: main.go [hunk 7]
```diff
@@ -86,7 +87,7 @@ func main() {
 	app.Run(os.Args)
 }
 
-func router(handler peopleHandler) http.Handler {
+func router(handler people.PeopleHandler) http.Handler {
```
Comment (tosan88) [inline main.go:90]: Assuming these will be used when handler is ready, can you please then remove the unnecessary TLS unsecure config for the http client? https://github.com/Financial-Times/v1-people-transformer/pull/1/files#diff-7ddfb3e035b42cd70649cc33393fe32cL113 It was accidentally left in v1-orgs-transformer as well :)

File: people/service.go (hunk 5 above)
Comment (carlosroman) [reply people/service.go]: That was just a random name I gave it while doing my refactor. It was my "new `init`" version of the `init` function that existed before.

File: main.go (hunk 1 above)
Comment (carlosroman) [reply main.go:75]: oh, that was just there to stop go complaining about stuff. This version won't go into any env and it is removed in the followup branch.

**Review by tosan88 — APPROVED** [review]

**Pull request merged**